### PR TITLE
#1750 Update task definitions for Comp & Pen

### PR DESCRIPTION
**Pull request by kalbfled**
# Description

Update task definitions for Comp & Pen by adding values for the DynamoDB table name and ARNs for the service ID and template ID parameters.

I made changes in all 3 of each environment's task definitions because the variable names were already there.  I think, for example, that the DynamoDB table name is only relevant to the "beat task definition", but I'm not going to verify that now.  We have #1711 to audit the task definitions for unnecessary variables.

issue #1750

## Type of change

Please check the relevant option(s).

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation changes only

## How Has This Been Tested?

How has this been tested? (e.g. Unit tests, Tested locally, Tested as a GitHub action, Depoyed to dev, etc)  
Please provide relevant information and / or links to demonstrate the functionality or fix. (e.g. screenshots, link to deployment, regression test results, etc)

- [x] Deployed to Dev

There's nothing else to test until these changes are deployed to Staging and Prod.

## Checklist

- [x] I have assigned myself to this PR
- [x] PR has an appropriate title: #9999 - What the thing does
- [x] PR has a detailed description, including links to specific documentation
- [x] I have added the [appropriate labels](https://github.com/department-of-veterans-affairs/notification-api/blob/master/.github/release.yaml) to the PR.
- [x] I did not remove any parts of the template, such as checkboxes even if they are not used
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to any documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have ensured the latest master is merged into my branch and all checks are green prior to review
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] The ticket is now moved into the DEV test column

### CODE REVIEW

**Review by MackHalliday — APPROVED** [review]
👍 🚀

**Review by MackHalliday — APPROVED** [review]

File: cd/application-deployment/dev/vaec-api-task-definition.json [hunk 1]
```diff
@@ -231,20 +231,20 @@
...
-        },
         {
           "name": "COMP_AND_PEN_DYNAMODB_NAME",
           "value": "dev-bip-msg-mpi-profile-participant-cache-table"
         }
       ],
       "secrets": [
+        {
+          "name": "COMP_AND_PEN_SERVICE_ID",
+          "value": "arn:aws-us-gov:ssm:us-gov-west-1:171875617347:parameter/dev/notification-api/comp-and-pen/service-id"
```
Comment (EvanParish) [inline cd/application-deployment/dev/vaec-api-task-definition.json:242]: Should we just remove these things from the API task definitions if they are only necessary in celery task definitions?

Comment (cris-oddball) [reply cd/application-deployment/dev/vaec-api-task-definition.json:242]: We had a previous convo about that, so I was surprised to see them all still being used in every task definition. What was the outcome of that previous convo?

Comment (cris-oddball) [reply cd/application-deployment/dev/vaec-api-task-definition.json:242]: Got it, we have another ticket to audit task defs. I'm fine with this going out as it stands then.

**Review by cris-oddball — APPROVED** [review]

**Review by EvanParish — APPROVED** [review]

**Pull request merged**